configure.ac

```diff
@@ -581,9 +581,18 @@ AS_CASE([${RKT_ACI_ARCH}],
 RKT_IF_HAS_FLAVOR([${RKT_STAGE1_FLAVORS}], [coreos],
                   [AS_CASE([${RKT_ACI_ARCH}],
                            [aarch64],
-                                    [RKT_STAGE1_COREOS_INTERPRETER="/usr/lib/ld-linux-aarch64.so.1"],
+                                    [RKT_STAGE1_INTERPRETER="/usr/lib/ld-linux-aarch64.so.1"],
                            [amd64],
-                                    [RKT_STAGE1_COREOS_INTERPRETER="/usr/lib/ld-linux-x86-64.so.2"])])
+                                    [RKT_STAGE1_INTERPRETER="/usr/lib/ld-linux-x86-64.so.2"])])
```

Comment:
Could the `.so` version ever change? If so, could you add a line in [update-coreos-stage1.md](https://github.com/coreos/rkt/blob/master/Documentation/devel/update-coreos-stage1.md) to say that configure.ac might need an update?